management-api-agent-common/src/main/java/com/datastax/mgmtapi/NodeOpsProvider.java

```diff
@@ -940,6 +941,22 @@ public String move(
     return submitJob("move", moveOperation, async);
   }
 
+  @Rpc(name = "reloadInternodeEncryptionTruststore")
+  public void reloadInternodeEncryptionTruststore() throws Exception {
+    ShimLoader.instance.get().reloadInternodeEncryptionTruststore();
+  }
+
+  @Rpc(name = "getInternodeEncryptionTruststoreIssuers")
+  public List<String> getEncryptionTruststoreIssuers() throws JsonProcessingException, Exception {
+    List<X509Certificate> issuers =
+        Arrays.stream(ShimLoader.instance.get().getEncryptionTruststoreIssuers())
+            .collect(Collectors.toList());
+    ObjectMapper objectMapper = new ObjectMapper();
+    return issuers.stream()
+        .map(i -> objectMapper.writeValueAsString(i))
```

Comment:
issue:
`writeValueAsString` throws checked exceptions, but you can't call methods that throw checked exceptions within a lambda expression, so this will need to be re-written

Reply:
Thanks for the feedback, this is now fixed.